core/src/test/java/com/datastax/oss/driver/internal/core/loadbalancing/DefaultLoadBalancingPolicyRequestTrackerTest.java

```diff
@@ -186,13 +186,13 @@ public void should_record_further_response_times_on_node_error() {
             node1,
             value -> {
               // values should rotate left (bubble up)
-              assertThat(value.get(0)).isEqualTo(456);
-              assertThat(value.get(1)).isEqualTo(789);
+              assertThat(value.oldest).isEqualTo(456);
+              assertThat(value.newest.getAsLong()).isEqualTo(789);
             })
-        .hasEntrySatisfying(node2, value -> assertThat(value.get(0)).isEqualTo(789))
+        .hasEntrySatisfying(node2, value -> assertThat(value.oldest).isEqualTo(789))
         .doesNotContainKey(node3);
     assertThat(policy.isResponseRateInsufficient(node1, nextNanoTime)).isFalse();
-    assertThat(policy.isResponseRateInsufficient(node2, nextNanoTime)).isTrue();
-    assertThat(policy.isResponseRateInsufficient(node3, nextNanoTime)).isTrue();
+    assertThat(policy.isResponseRateInsufficient(node2, nextNanoTime)).isFalse();
+    assertThat(policy.isResponseRateInsufficient(node3, nextNanoTime)).isFalse();
```

Comment:
Semi-nit: after these changes we no longer have any test case which tests assumptions about when isResponseRateInsufficient() will return true.  We only had a few before so this isn't the end of the world... and it's not as if that method isn't reasonably well exercised by the logic here.  But I am just a bit concerned that we should have a test case to confirm that this method will return true when we create cases where it should return true.